site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
I am not an expert in npm, how is `save` different from `save-dev`?
Here is my understanding: it is a dev dependency if this package only distribute js files, it would be a dependency if this package distribute both ml and js files.
Are you familiar how typescript works, different versions of typescript may conflict, is `typescript` dev-dependency or dependency?